.prettierignore

```diff
@@ -1,8 +1,8 @@
-.turbo
-.github
 dist/
 node_modules/
 **/*.css
+**/*.scss
 **/*.md
 **/*.json
-yarn.lock
\ No newline at end of file
+yarn.lock
+.*
```

Comment:
Maybe I understand it wrongly, but doesn't that ignore every file?

Reply:
(The `.*`, I mean)

Reply:
You're thinking of `./*`. `.*` ignores all of the config files at repo root. Prettier was tripping over trying to parse the .gitignore file so I had to include this

Reply:
Ah, fair enough, thanks for helping me understand it!